common/js/rechat.js

```diff
@@ -428,6 +476,21 @@ ReChat.Playback.prototype._formatJtvMessage = function(messageData) {
   return this._formatSystemMessage($.extend(messageData, { message: message }), classification);
 };
 
+ReChat.Playback.prototype._getSubscriberUrl = function() {
+  var that = this;
+  $.ajax({
```

Comment:
`$.ajax` does not work in Safari. Use `ReChat.get` instead (see `ReChat.Playback.prototype._loadMessages` for a usage example)